packages/colors/src/colors.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const black = '#000000';
+export const black100 = black;
```

Comment:
Was thinking about that, but then was thinking about how folks would use `black` as an export and would assume that `black` refers to the color itself versus the swatch. Since the swatches for them would only have one value, it felt a little off when I made that change.